portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,20 @@
 /**
  * Adds a random greeting to the page.
  */
+
 function addRandomGreeting() {
-  const greetings =
+  /*const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  greetingContainer.innerText = greeting;*/
+  let a = new ProjectContainer();
 }
+
+let nextItem = () =>{
```

Comment:
Maybe I'm missing something but it's not obvious to me what this function's doing.